### PR TITLE
Rest: premiseInfo: No premise,  return premise null - https://www.pivota...

### PR DESCRIPTION
**Pull request by RamalingamThangavel**
...ltracker.com/story/show/93665504

### CODE REVIEW

**Comment by fiallega** [comment]
Nice first commit

**Comment by RamalingamThangavel** [comment]
@fiallega hey can you review the code

File: src/main/resources/xsl/CmSSvcPremiseServiceAndMeterInfoService.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<CmSSvcPremiseServiceAndMeterInfo>
```
Comment (fiallega) [inline src/main/resources/xsl/CmSSvcPremiseServiceAndMeterInfoService.xml]: I think I would call the file something inline to what we have been using: CmSSvcPremiseServiceAndMeterInfoServiceResponse.xml

File: src/main/resources/xsl/CmSSvcPremiseServiceAndMeterInfoServiceResponseSuccess.xsl [hunk 2]
```diff
@@ -18,35 +18,39 @@
...
-                    </pool>
-                    <location>
-                        <latitude><xsl:value-of select="normalize-space(//latitude)"/></latitude>
-                        <longitude><xsl:value-of select="normalize-space(//longitude)"/></longitude>
-                    </location>
-                    <folio><xsl:value-of select="normalize-space(//premise/folio)"/></folio>
-                    <municipalityCode><xsl:value-of select="normalize-space(//premise/municipalityCode)"/></municipalityCode>
-                    <districtCode><xsl:value-of select="normalize-space(//premise/districtCode)"/></districtCode>
-                     </premise>
+                <xsl:apply-templates select="//premise" />
```
Comment (ramonjsantiago) [inline src/main/resources/xsl/CmSSvcPremiseServiceAndMeterInfoServiceResponseSuccess.xsl]: Can I get an account # that has no premise?

On Mon, May 4, 2015 at 10:45 AM, Jorge L Fiallega <notifications@github.com>
wrote:

> In
> src/main/resources/xsl/CmSSvcPremiseServiceAndMeterInfoServiceResponseSuccess.xsl
> <https://github.com/miamidade/wasd-self-service/pull/23#discussion_r29589948>
> :
>
> > -                           </address>
> > -
> > -                    <pool>
> > -                        <onPropertyRecord><xsl:value-of select="normalize-space(//premise/pool/onPropertyRecord)"/></onPropertyRecord>
> > -                        <size><xsl:value-of select="normalize-space(//premise/pool/size)"/></size>
> > -                        <type><xsl:value-of select="normalize-space(//premise/pool/type)"/></type>
> > -                    </pool>
> > -                    <location>
> > -                        <latitude><xsl:value-of select="normalize-space(//latitude)"/></latitude>
> > -                        <longitude><xsl:value-of select="normalize-space(//longitude)"/></longitude>
> > -                    </location>
> > -                    <folio><xsl:value-of select="normalize-space(//premise/folio)"/></folio>
> > -                    <municipalityCode><xsl:value-of select="normalize-space(//premise/municipalityCode)"/></municipalityCode>
> > -                    <districtCode><xsl:value-of select="normalize-space(//premise/districtCode)"/></districtCode>
> > -                     </premise>
> > +                <xsl:apply-templates select="//premise" />
>
> I am not sure this is going to show a premise node if one does not come
> from CCB
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/miamidade/wasd-self-service/pull/23/files#r29589948>.
>



-- 
rjs

**Comment by RamalingamThangavel** [comment]
@fiallega Please review.

**Pull request merged**